compiler/ballerina-parser/src/main/java/io/ballerinalang/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -284,6 +285,9 @@
     private static final ParserRuleContext[] LET_VAR_DECL_START =
             { ParserRuleContext.TYPE_DESCRIPTOR, ParserRuleContext.ANNOTATIONS };
 
+    private static final ParserRuleContext[] STREAM_TYPE_PARAMS =
```

Comment:
good to be more explicit like: `stream-type-first-param-rhs`

Reply:
@SupunS changed it.